src/component/legend/legendAction.ts

```diff
@@ -37,6 +37,9 @@ function legendSelectActionHandler(methodName, payload, ecModel) {
         else if (methodName === 'allSelect' || methodName === 'inverseSelect') {
             legendModel[methodName]();
         }
+        else if (methodName === 'legendMouseover' || methodName === 'legendMouseout') {
+            isSelected = legendModel.isSelected(payload.name);
```

Comment:
This is perhaps no longer needed. You make merge the current master code and have a double check.

Reply:
You should update the `LegendSelectMethodNames` after merging from master.